pkg/kube/resourcerequirements/resource_requirements.go

```diff
@@ -0,0 +1,49 @@
+package resourcerequirements
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	resourceMemory = "memory"
+	resourceCpu    = "cpu"
+)
+
+// New returns a new corev1.ResourceRequirements with the specified arguments, and an error
+// which indicates if there was a problem parsing the input
+func New(limitsCpu, limitsMemory, requestsCpu, requestsMemory string) (corev1.ResourceRequirements, error) {
+	limits, err := buildResourceList(limitsCpu, limitsMemory)
+	if err != nil {
+		return corev1.ResourceRequirements{}, err
+	}
+
+	requests, err := buildResourceList(requestsCpu, requestsMemory)
+	if err != nil {
+		return corev1.ResourceRequirements{}, err
+	}
+	return corev1.ResourceRequirements{
+		Limits:   limits,
+		Requests: requests,
+	}, nil
+}
+
+// Default returns the default resource requirements for a container
+func Default() (corev1.ResourceRequirements, error) {
+	return New("1.0", "500M", "0.5", "400M")
```

Comment:
Discard the returning errors of this function, `Default` should not fail as the values are known

Reply:
Possibly add a unit test that checks that `Default()` return these values so there's not way we introduce errors in the `Default` impl.

I mention this because I see a few lines up there checking the resulting value of the Default function

Reply:
good point about removing the error in this case 👍